run_sdk_container

```diff
@@ -150,6 +151,7 @@ if [[ -z ${stat} ]] ; then
         --network host
         -e SDK_USER_ID="$(id -u)"
         -e SDK_GROUP_ID="$(id -g)"
+	-e MODULE_SIGNING_KEY_DIR="/tmp/$(uuidgen)"
```

Comment:
The whitespace is off here, but I think it might be better to set this in sdk_entry.sh
anyway.